### PR TITLE
Doc updates for vSphere 6.7 update 1

### PR DESCRIPTION
**Pull request by stuclem**
Added requirement to use 1.4.3 on vSphere 6.7u1 in https://github.com/vmware/vic-product/commit/9b172037ef57c7e2bb4db5eed71db785a17c87f5.

Fixes https://github.com/vmware/vic-product/issues/1999

### CODE REVIEW

**Comment by stuclem** [comment]
https://github.com/vmware/vic-product/pull/2130/commits/271d8689d313621e7b6ee97d7afbf3e53e986e5a fixes https://github.com/vmware/vic-product/issues/1945.

**Comment by stuclem** [comment]
@renmaosheng and @lgayatri can you please review this PR? Thanks!

**Comment by renmaosheng** [comment]
I think the reason we ask users to use VIC 1.4.3 for vSphere 6.7 u1, not just the html5 client, but also this is the version we have validated for vSphere 6.7 u1.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ You can deploy multiple vSphere Integrated Containers appliances to the same vCe
 If you are deploying vSphere Integrated Containers 1.4.3 or later, by default the vSphere Integrated Containers plug-in for the vSphere Client is installed automatically. The installer installs the following plug-ins:
 
 - A plug-in for the HTML5 vSphere Client on vCenter Server 6.5 and 6.7. The HTML5 plug-in allows you to to deploy and interact with virtual container hosts (VCHs) directly in the vSphere Client.
+
+  **IMPORTANT**: If you use vSphere 6.7 update 1 or later, you must use vSphere Integrated Containers 1.4.3 or later. Due to significant changes in the HTML5 vSphere Client in version 6.7 update 1, previous versions of the vSphere Integrated Containers plug-in for the vSphere Client do not work with that version.
```
Comment (renmaosheng) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Could we also mention VIC 1.4.3 is the first version we did the compatible test against vSphere 6.7 u1.

**Comment by stuclem** [comment]
Thanks @renmaosheng. I made the change in https://github.com/vmware/vic-product/pull/2130/commits/cac545f98456204f9bd2c63aa5f0e43dcd45f163:

**IMPORTANT**: If you use vSphere 6.7 update 1 or later, you must use vSphere Integrated Containers 1.4.3 or later. Due to significant changes in the HTML5 vSphere Client in version 6.7 update 1, previous versions of the vSphere Integrated Containers plug-in for the vSphere Client might not work with that version. Only version 1.4.3 has been verified with vSphere 6.7 update 1.

**Pull request merged**